actions/sharepoint.js

```diff
@@ -363,6 +363,30 @@ async function getExcelTable(excelPath, tableName) {
     return [];
 }
 
+async function deleteFloodgateDir(fgRootFolder) {
+    const logger = getAioLogger();
+    logger.info('Deleting content started.');
+    const baseURI = `${appConfig.getConfig().fgSite}/drive/root:${fgRootFolder}`;
+    let deleteSuccess = false;
+
+    const { fgDirPattern } = appConfig.getConfig();
+    const fgRegExp = new RegExp(fgDirPattern);
+    logger.info(fgRegExp);
+    if (fgRegExp.test(baseURI)) {
+        logger.info(`Deleting the folder ${baseURI} `);
+        const temp = '/drafts/nsivakum/trial';
```

Comment:
@Nethraasivakumar Was this validated in your AIO workspace?  Could you share the action URL to try it out? Also could you change this path to `/temp`.